### DISCUSSION TITLE
essay/2024/05/03/coffee-server

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#209] [Comments]
# essay/2024/05/03/coffee-server

집에서 서버를 6년 동안 운영했다는 것이 많은 분들에게 신기하게 들렸나 봅니다. 이해합니다. 옛 개발자 동료들도 이야기를 들으면 대부분 놀랐으니까.

https://jeho.page/essay/2024/05/03/coffee-server.html

**Comment by developedbyDK** [comment]
리바이스 청자켓 힙하고 멋있네요 ㅎㅎ 
팟케스트 보면서 돈많이버셔서 하나 사셨나보다 생각했는데 30년 전 것이었군요!

**Reply by BenjaminKim** [reply]
하하하. 몇년 동안 추리링만 입고 다녔는데... 아침에 뭘 입고 가야하나 고민하다가 꺼내 입었습니다.
30년 만에 입었네요.

**Comment by brokim-ensol** [comment]
처음으로 사이드 프로젝트를 시작하려다가 가성비 있게 구축하는 방법에 대해 고민 중에 있습니다.
그러다가 우연히 긱뉴스에서 발표 자료를 보고 오게 되었네요. 생생한 경험 감사드립니다.

**Reply by BenjaminKim** [reply]
도움이 되었다면 기쁜 일입니다!
